fizzBuzzRedux.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Fizz Buzz Redux</title>
+</head>
+<body>
+  <form id="inputForm" action="javascript:onButtonClick()">
```

Comment:
While this technically meets the goals of the assignment, one of the things we were looking for was event handling. This approach is pretty dated and has been phased out in favor of either library specific (`ng-submit`) or unobtrustive (event handler) methods.

See the [plethora of answers here](http://stackoverflow.com/questions/5871640/why-is-using-onclick-in-html-a-bad-practice) for more information. The tl;dr - you're running your `action` value through `eval`, which is considered harmful.